--- admin/acinclude.m4.in~	Wed May 16 14:39:19 2001
+++ admin/acinclude.m4.in	Tue May 22 04:32:31 2001
@@ -3566,14 +3566,10 @@
   dnl Note: this test is very weak
   kde_python_link_found=no
   KDE_TRY_LINK_PYTHON(normal)
-  KDE_TRY_LINK_PYTHON(m, -lm)
-  KDE_TRY_LINK_PYTHON(pthread, $LIBPTHREAD)
-  KDE_TRY_LINK_PYTHON(tcl, -ltcl)
-  KDE_TRY_LINK_PYTHON(m_and_thread, [$LIBPTHREAD -lm])
-  KDE_TRY_LINK_PYTHON(m_and_thread_and_util, [$LIBPTHREAD -lm -lutil])
-  KDE_TRY_LINK_PYTHON(m_and_thread_and_db3, [$LIBPTHREAD -lm -ldb-3 -lutil])
-  KDE_TRY_LINK_PYTHON(pthread_and_db3, [$LIBPTHREAD -ldb-3])
-  KDE_TRY_LINK_PYTHON(m_and_thread_and_db, [$LIBPTHREAD -lm -ldb -ltermcap -lutil])
+  KDE_TRY_LINK_PYTHON(gentoo1, [$LIBPTHREAD -lm -ldb-3.2 -lreadline -lcrypt -lz -lncurses -lutil])
+  KDE_TRY_LINK_PYTHON(gentoo2, [$LIBPTHREAD -lm -lreadline -lcrypt -lz -lncurses -lutil])
+  KDE_TRY_LINK_PYTHON(gentoo3, [$LIBPTHREAD -lm -ldb-3.2 -lcrypt -lz -lncurses -lutil])
+  KDE_TRY_LINK_PYTHON(gentoo4, [$LIBPTHREAD -lm -lcrypt -lz -lncurses -lutil])
   KDE_TRY_LINK_PYTHON(m_and_thread_and_db_special, [$LIBPTHREAD -lm -ldb -lutil], [],
 	[AC_MSG_WARN([it seems, Python depends on another library.
     Pleae use \"make LIBPYTHON='-lpython$version -lotherlib'\" to fix this
